packages/remediations/src/NewRemediationWizard.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component, Fragment, createContext } from 'react';
+import componentMapper from '@data-driven-forms/pf4-component-mapper/dist/cjs/component-mapper';
+import FormRenderer from '@data-driven-forms/react-form-renderer/dist/cjs/form-renderer';
+import Pf4FormTemplate from '@data-driven-forms/pf4-component-mapper/dist/cjs/form-template';
+import schemaBuilder from './schema';
+import Deferred from '@redhat-cloud-services/frontend-components-utilities/files/Deffered';
+
+const RemediationWizardContext = createContext({
+    success: false,
+    submitting: false,
+    error: undefined
+});
+
+class RemediationWizard extends Component {
+
+    state = {
+        open: false,
+        schema: {}
+    }
+
+    constructor() {
+        super();
+        this.container = React.createRef(document.createElement('div'));
+    }
+
+    setOpen = open => {
+        this.setState({ open });
+    }
+
+    getFormTemplate = (props) => <Pf4FormTemplate {...props} showFormControls={false} />;
+
+    openWizard = () => {
+        const deferred = new Deferred();
+        this.setState({
+            open: true,
+
+            schema: schemaBuilder(this.container.current),
+
+            wizardContextValue: {
+                success: false,
+                submitting: false,
+                error: undefined,
+                hideForm: false
+            }
+        });
+
+        // TO DO - load remediations
+        // TO DO - load resolutions
+
+        return deferred.promise;
+    }
+
+    setWizardError = (error) => this.setState({ wizardContextValue: { ...this.state.wizardContextValue, error } });
+
+    setWizardSuccess = (success) => this.setState({ wizardContextValue: { ...this.state.wizardContextValue, success } });
+
+    setHideForm = (hideForm) => this.setState({ wizardContextValue: { ...this.state.wizardContextValue, hideForm } });
+
+    closeWizard = () => {
+        this.setOpen(false);
+    }
+
+    render () {
+
+        return (
+            this.state.open !== false ?
+                <RemediationWizardContext.Provider
+                    value={{ ...this.state.wizardContextValue,
+                        setWizardError: this.setWizardError,
+                        setWizardSuccess: this.setWizardSuccess,
+                        setHideForm: this.setHideForm }}>
+                    <FormRenderer
+                        schema={this.state.schema}
+                        container={this.container}
+                        subscription={{ values: true }}
+                        FormTemplate={this.getFormTemplate}
+                        componentMapper={{ ...componentMapper }}
```

Comment:
there is no need to copy the component mapper object. It's also better for performance, to only add a minimal set of components to the mapper. Instead of passing the whole object, it's better to directly import components (text fields, radio buttons, etc.) and create a mapper object from those.